amaranth-chrome-ext/test/AmaranthUtil.test.js

```diff
@@ -0,0 +1,37 @@
+const AmaranthUtil = require('../src/AmaranthUtil');
+
+// Test AmaranthUtil.removeSpecialCharacters
```

Comment:
I used the comments in this file to demarcate where certain functions are tested. I found it difficult to tell which function was being tested where since they all start with `test(...`. Should I make this more obvious? Or just remove altogether

Reply:
Will add name of function being tested to test description

Reply:
Fixed in https://github.com/googleinterns/amaranth/pull/44/commits/57599a151e27079431950dcd971f2fb4fec5707a